packages/table/preview/index.tsx

```diff
@@ -377,11 +378,11 @@ ReactDOM.render(
             return <Cell intent={rowIndex as Intent}>{Utils.toBase26Alpha(columnIndex) + (rowIndex + 1)}</Cell>;
         },
     }, {
-        isColumnResizable: false,
-        isRowResizable: false,
-        selectionModes: SelectionModes.NONE,
-    }),
-    document.getElementById("table-1")
+            isColumnResizable: false,
+            isRowResizable: false,
+            selectionModes: SelectionModes.NONE,
+        }),
```

Comment:
sure, i didn't look closely at the formatting. maybe i'll just revert, it's not super relevant.